src/style.css

```diff
@@ -0,0 +1,177 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap');
+
+:root {
+  --font-weight: 300;
+  --font: 'Poppins', sans-serif;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  margin: 0;
+  background-color: white;
+  font-family: var(--font);
+  font-weight: var(--font-weight);
+}
+
+main {
+  text-align: center;
+  padding-top: 200px;
+}
+
+ul {
+  list-style: none;
+}
+
+input {
+  width: max-content;
+  border-top-style: hidden;
+  border-right-style: hidden;
+  border-left-style: hidden;
+  border-bottom-style: hidden;
+}
+
+input:focus,
+textarea:focus,
+select:focus {
+  outline: none;
+}
+
+.line {
+  border-bottom: 1px solid rgb(204, 204, 204);
+}
+
+.head-title {
+  font-style: normal;
+  margin-bottom: 0;
+  margin-top: 0;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.svg-inline--fa {
+  font-size: 12px;
+  color: rgb(160, 160, 160);
+  transition: 0.3s;
+}
+
+.svg-inline--fa:hover {
+  color: rgb(63, 62, 62);
+  cursor: pointer;
+  font-size: 12px;
+}
+
+.fa-trash-can:hover {
+  color: rgb(248, 68, 68);
+  cursor: pointer;
+  font-size: 12px;
+}
+
+.fa-ellipsis-vertical:hover {
+  cursor: move;
+}
```

Comment:
- [ ] What if we remove `:hove` and simply use `.fa-ellipsis-vertical {
  cursor: move;
}` to keep the code YAGNI